src/literals.py

```diff
@@ -4,10 +4,10 @@
 
 """Collection of global literals for the charm."""
 
-OPENSEARCH_DASHBOARDS_SNAP_REVISION = "24"
+OPENSEARCH_DASHBOARDS_SNAP_REVISION = "1"
```

Comment:
Should this be open search or wazuh?

Reply:
We could rename everything, but I'm trying to diverge as less as possible from upstream, so I'm keeping all the code as it is and change only the indispensable